### DISCUSSION TITLE
AGN parameters in detail

### DISCUSSION THREAD

**Discussion by alessandropeca** [mboquien/cigale#50] [Q&A]
Hi,

Is there some detailed documentation on the physical parameters that CIGALE computes?
I did not find the details in the manual. For example, the luminosity of `agn.polar_dust_luminosity, agn.torus_dust_luminosity, agn.disk_luminosity, agn. luminosity` in which bands (energy or wavelengths) are computed?

**Comment by Guang91** [comment]
They're calculated over a very wide wavelength range from UV to far-IR, not for a specific band. You can consider them as "total" luminosity from different components.

**Reply by Ecekilerci** [reply]
Hi, I have a related question. I have X-ray data in the SED fitting starting from 0.5 keV. After the fit I want to have the classical bolometric luminosity from the best fit results. Is it ok just to add agn. luminosity or best.agn.disk_luminosity and X-ray luminosity? 
Actually, it would be very hand if CIGALE could give the total AGN Lbol.

**Comment by Guang91** [comment]
Yes, you could use bolometric luminosity = X-ray luminosity +
agn.luminosity.

**Reply by Ecekilerci** [reply]
Thanks!

**Comment by MTHGET** [comment]
Hi, I also have a similar question. I am using the AGN fraction (frac_AGN), which I believe is computed as frac_AGN = L_AGN,dust/(L_AGN,dust + L_AGN,galaxy). Now I am trying to figure out the wavelength ranges over which each luminosity is integrated. Based on your response above, are they individually computed over 8-1000um?

**Comment by Guang91** [comment]
It includes all wavelengths of the dust luminosities (though 8-1000um
actually covers most of the energy).

A similar discussion can be found at
https://github.com/mboquien/cigale/discussions/163
